src/NServiceBus.Core/Licensing/LicenseManager.cs

```diff
@@ -129,5 +193,6 @@ void PromptUserForLicenseIfTrialHasExpired()
         License license;
 
         static ILog Logger = LogManager.GetLogger(typeof(LicenseManager));
+        static HttpClient httpClient = new HttpClient();
```

Comment:
How many times do we do the call home call? Does it make sense to keep the http client statically around forever?